tools/rules.yml

```diff
@@ -1,57 +0,0 @@
-subject: namespace:ns
```

Comment:
nit: is there a replacement for this set-up in tools/?

Reply:
I don't think so. What is this? There is no reference to this file in any repo and hasn't been touched in over a year

Reply:
Do you need this?